dev to release/v0.1.9

thomashebrard opened the pull request.

antoinepipelex reviewed (approved):
ok

antoinepipelex reviewed (commented):
I have read the CLA Document and I hereby sign the CLA

lchoquel commented:
recheck

lchoquel committed:
Draft pipeline to Write Tweet

lchoquel committed:
Tweet writer working, basic

lchoquel committed:
Moved wip pipelines. Overdue proofread of Contributing doc.

lchoquel committed:
Cleanup docs and settings and stuff

lchoquel committed:
using Pipelex v0.2.11

lchoquel committed:
Support Python 3.10 with fixes for Self and StrEnum

lchoquel committed:
GHA for python versions 10 thru 13
no more google credentials in UT

lchoquel committed:
Moved stuff

lchoquel committed:
Improve tweet sample

lchoquel committed:
Merge branch 'refactor/Support-Python-Versions' into feature/Write-Tweet

# Conflicts:
#	uv.lock

lchoquel committed:
Merge pull request #9 from Pipelex/feature/Write-Tweet

- sample to improve a tweet
- various fixes and improvements in docs and code

lchoquel committed:
### 📝 Description

- rename Mission to Pipeline
- enable to start a pipeline and let in run in the background, getting it's run id, but not waiting for the output
- Makefile: avoid defaulting pytest to verbose. Setup target `make test-xdist` = Run unit tests with xdist, make it the default for shorthand `make t`. The old make t is now make tp (test-with-prints)

### 🔄 Type of Change
- [ ] 🐛 Bug fix
- [X] ✨ New feature
- [ ] 💥 Breaking change
- [ ] 📚 Documentation update
- [X] 🧹 Code refactor
- [ ] ⚡ Performance improvement
- [ ] ✅ Test update

lchoquel committed:
use pipelex branch dev

lchoquel committed:
Use pipelex dev for real

lchoquel committed:
Merge pull request #10 from Pipelex/refactor/Rename-Pipeline

Refactor/rename pipeline

thomashebrard committed:
Fix cli based on the pipelex==0.2.12 cli (#11)

* fix/pipelex-0.2.12

* lock

thomashebrard committed:
bump pipelex-cookbook version

thomashebrard committed:
add changelog
